### DISCUSSION TITLE
Disable commenting highlight

### DISCUSSION THREAD

**Discussion by jin-ahn** [catppuccin/nvim#498] [Q&A] [answered]
How do I disable highlighting of comments?
Below is my config, and you can see there's a comment that the theme highlights, I just want my comments to be grayed out, not highlighted

![image](https://github.com/catppuccin/nvim/assets/66384196/e930e7eb-796e-4287-aba6-0d1f2af6f912)

**Comment by EdenEast** [comment, marked as answer]
If you are using tmux you have to fix the color codes https://github.com/catppuccin/nvim/issues/428#issuecomment-1461864762 the issue is with your terminal.
